services/drupal/web/modules/custom/epa_workflow/epa_workflow.module

```diff
@@ -254,6 +254,34 @@ function epa_workflow_form_alter(array &$form, FormStateInterface $form_state, $
         $form['node_edit_link'] = $edit_link->toRenderable();
       }
     }
+
+    // Want to display the `field_review_deadline` date
+    // on the moderation block only on published content.
+    if (isset($node) && $node->isPublished()) {
+
+      $timezone = date_default_timezone_get();
+      $date_formatter = \Drupal::service('date.formatter');
+
+      $review_deadline = $date_formatter->format(
+        $node->field_review_deadline->date->getTimestamp(), 'custom', 'F d, Y h:i a T', $timezone
```

Comment:
Let's guard against the field_review_deadline possibly not being set.  It really should always be set for published nodes, but if some policies fail to occur properly then it's possible for it to be unset and we don't want to throw an error, we should just not display the field.

Reply:
Added a check for `field_review_deadline` & `field_review_deadline->date`